### PR TITLE
Update the logging namespace - qemu (u)

### PR DESCRIPTION
**Pull request by luckyh**
Issue: https://github.com/autotest/tp-qemu/issues/3048

Signed-off-by: Xu Han <xuhan@redhat.com>

### CODE REVIEW

File: qemu/tests/usb_basic_check.py [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
+LOG_JOB = logging.getLogger('avocado.test')
+
 
 @error_context.context_aware
 def check_usb_device_monitor(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
 
-    error_context.context("Verify USB device in monitor.", logging.info)
+    error_context.context("Verify USB device in monitor.", LOG_JOB.info)
```
Comment (yanan-fu) [inline qemu/tests/usb_basic_check.py:17]: Replace all `LOG_JOB` with `test.log` is safe for this file.

File: qemu/tests/usb_basic_check.py [hunk 2]
```diff
@@ -126,14 +128,14 @@ def _check_dev():
...
-    error_context.context("Try to log into guest.", logging.info)
+    error_context.context("Try to log into guest.", test.log.info)
     session = vm.wait_for_login(timeout=timeout)
 
     error_context.context("Verify device(s) before rebooting.")
     _check_dev()
 
     if params.get("reboot_method"):
-        error_context.context("Reboot guest.", logging.info)
+        error_context.context("Reboot guest.", test.log.info)
```
Comment (yanan-fu) [inline qemu/tests/usb_basic_check.py:138]: I want to comment L143, not this line.   Could you fix the issue together with this patch, add `test.log.info` in L143 ?

File: qemu/tests/usb_storage.py [hunk 3]
```diff
@@ -124,14 +123,14 @@ def _get_output():
     @error_context.context_aware
     def _check_serial_option(serial, regex_str, expect_str):
         error_context.context("Set serial option to '%s'" % serial,
-                              logging.info)
+                              test.log.info)
```
Comment (yanan-fu) [inline qemu/tests/usb_storage.py:126]: L101 miss `test.log.info` for `error_context.context`.  Would you mind to update it together ?

File: qemu/tests/usb_basic_check.py (hunk 2 above)
Comment (luckyh) [reply qemu/tests/usb_basic_check.py:135]: After our discussion, will skip this suggestion as it seems to be the expected behavior, thanks.

File: qemu/tests/usb_storage.py (hunk 3 above)
Comment (luckyh) [reply qemu/tests/usb_storage.py:126]: The same as the above.

**Comment by luckyh** [comment]
@yanan-fu have updated per your suggestions, thanks.

**Review by yanan-fu — APPROVED** [review]
LGTM

Acked-by: Yanan Fu <yfu@redhat.com>

**Pull request merged**